### PR TITLE
NAV - fix top left corner alignment

### PR DESCRIPTION
**Pull request by kobryan0619**
This:

- Aligns content to the top left corner of the page for V2
- Removes the header on the "Progress" page when in V2
- Ensures the content on the "empty section" page is aligned in V2
- Does NOT refine the header spacing wise in V2 (theres likily more spacing work to do in a follow up PR)
- Does NOT intentionally put the v1 v2 toggle for progress anywhere (it is there now, but this was not a design choice made by design - that will also need to be decided in a later PR)
- It does NOT touch the headline on the "Settings" page

Here's what the old nav looks like after this change (no changes are intended, this should just verify everything looks right)

https://github.com/user-attachments/assets/bcd5c722-9202-4899-8fbb-527444e364fd

And here is what it looks like with the V2 nav:

https://github.com/user-attachments/assets/12b78ae1-944a-4835-a8ce-bbdc9f84de0b

## Links

[Figma](https://www.figma.com/design/5ILfiJkPQpjskBwksMDeZl/Teacher-Dashboard?node-id=4121-18467&m=dev)
[Ticket](https://codedotorg.atlassian.net/jira/software/projects/TEACH/boards/65?selectedIssue=TEACH-1267)

### CODE REVIEW

**Comment by MollyPeredo** [comment]
It's so exciting to see this coming together! Great work, Kaitie.

This might not have anything to do with this PR, but is there any way to avoid the Progress (beta) heading loading in a non-brand font initially and then switching to the brand font when the page content loads? This happens twice in the video showing the old nav.

For the video showing the new nav, are we planning to remove the "Edit Section Details" heading from the Settings page in a different PR? @lfryemason and I chatted about this yesterday, but I'm not sure what the plan for implementation was.

**Comment by kobryan0619** [comment]
> This might not have anything to do with this PR, but is there any way to avoid the Progress (beta) heading loading in a non-brand font initially and then switching to the brand font when the page content loads? This happens twice in the video showing the old nav.

To me that happens all the time on my local environment for all sorts of things.  I think it is something to watch when we merge in, but I am not worried about that in this PR.

> For the video showing the new nav, are we planning to remove the "Edit Section Details" heading from the Settings page in a different PR? @lfryemason and I chatted about this yesterday, but I'm not sure what the plan for implementation was.

I do feel like that is beyond the scope of this ticket.  That being said, in this ticket, I did remove the "Progress" header in this PR somewhat in hopes of getting Liam's feedback on that approach for the header.  Once we get on the same page for that approach, I think it is reasonable to tackle that next.

File: apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx [hunk 1]
```diff
@@ -27,6 +28,7 @@ function SectionProgressSelector({
   progressTableV2ClosedBeta,
   sectionId,
   hasSeenProgressTableInvite,
+  isInV2Navigaton,
```
Comment (lfryemason) [inline apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx]:
```suggestion
  isInV1Navigaton = false,
```
Optional nit: if we end up keeping this, then having the v2 as the default will make logic deletion easier when we are able to delete v1 code.

File: apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx [hunk 2]
```diff
@@ -142,14 +144,19 @@ function SectionProgressSelector({
...
+      )}
+    >
       {displayV2 && (
         <ProgressBanners hasJustSwitchedToV2={hasJustToggledViews} />
       )}
       {toggleV1OrV2Link()}
 
       {displayV2 ? (
-        <SectionProgressV2 />
+        <SectionProgressV2 hideTopHeading={isInV2Navigaton} />
```
Comment (lfryemason) [inline apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx]: There was a [discussion](https://codedotorg.slack.com/archives/C06ERUABG01/p1724781990022289) yesterday about whether we can remove the header from the v1 entirely. Hopefully we can just remove this logic 🤞

**Review by lfryemason — COMMENTED** [review]
Just assigned these two tickets that are also fixed by this PR: 
https://codedotorg.atlassian.net/browse/TEACH-1277
https://codedotorg.atlassian.net/browse/TEACH-1278

File: apps/src/templates/teacherNavigation/teacher-navigation.module.scss [hunk 3]
```diff
@@ -9,13 +9,13 @@
...
   flex-grow: 1;
-  padding: 20px;
+  padding: 20px 48px 20px 38px;
   overflow-y: auto;
 }
 
 .widthLockedPage {
   max-width: $content-width;
-  padding: 10px 64px 64px 64px;
+  padding: 10px 64px 64px 0;
```
Comment (lfryemason) [inline apps/src/templates/teacherNavigation/teacher-navigation.module.scss]: What do you think about removing the right padding as well? Currently these pages have 48px from the pageWithHeader and 64px from widthLockedPage

File: apps/src/templates/teacherNavigation/teacher-navigation.module.scss [hunk 4]
```diff
@@ -9,13 +9,13 @@
 
 .pageWithHeader {
   flex-grow: 1;
-  padding: 20px;
+  padding: 20px 48px 20px 38px;
```
Comment (lfryemason) [inline apps/src/templates/teacherNavigation/teacher-navigation.module.scss]: I'm tempted to add more padding at the bottom here. Most pages already have some, but for others only 20px might make it cramped. What do you think?

**Review by lfryemason — APPROVED** [review]
Looks fantastic! Discussion came out to keep the logic, so just a few small optional nits.

File: apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx (hunk 2 above)
Comment (lfryemason) [reply apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx]: Unfortunately we can't remove it from the v1 yet :(

File: apps/src/templates/teacherNavigation/teacher-navigation.module.scss (hunk 4 above)
Comment (kobryan0619) [reply apps/src/templates/teacherNavigation/teacher-navigation.module.scss]: I added a little, which I think will help, but I am also trusting that to work itself out during a future bug bash.

**Comment by tess323** [comment]
Wowwowwo looks so good!

**Pull request merged**